additional-material/translations/reverting-a-commit.it.md

```diff
@@ -1,41 +1,32 @@
 # Ripristino (o annullamento) di un commit (Revert)
 
-Ripristinare un commit significa semplicemente creare un commit nuovo di zecca che annulla tutti i cambiamenti fatti nel precedente. E' come fare un ```CTRL + Z ``` su git.  
+Ripristinare un commit significa semplicemente creare un commit nuovo di zecca che annulla tutti i cambiamenti fatti nel precedente. E' come fare un ```CTRL + Z ``` su git.
 
-Il ripristino è fatto semplicemente in git perché ogni commit che tu invii al tuo repository remoto ha una chiave alfanumerica univoca collegata, conosciuta come SHA (Secure Hash Algorithm).  
-Questo significa che puoi ripristinare qualsiasi commit fintanto che hai lo SHA.  
-Però devi stare attento a ripristinare ordinatamente per non rovinare il tuo repository.  
+Git semplifica effettuare il ripristino perché ogni commit inviato al tuo repository remoto ha una chiave alfanumerica univoca collegata, conosciuta come SHA (Secure Hash Algorithm). Questo significa che puoi ripristinare qualsiasi commit fintanto che hai lo SHA. Devi però stare attento a ripristinare ordinatamente per non rovinare il tuo repository.
```

Comment:
I believe this is not correct 'Git semplifica effettuare'

Reply:
It's technically correct, but it might sound better replacing "effettuare", let me know. 
```suggestion
Git semplifica eseguire il ripristino perché ogni commit inviato al tuo repository remoto ha una chiave alfanumerica univoca collegata, conosciuta come SHA (Secure Hash Algorithm). Questo significa che puoi ripristinare qualsiasi commit fintanto che hai lo SHA. Devi però stare attento a ripristinare ordinatamente per non rovinare il tuo repository.
```